src/soc/ibm/power9/istep_14_2.c

```diff
@@ -0,0 +1,51 @@
+#include <cpu/power/istep_14.h>
+#include <cpu/power/istep_13.h>
+
+static void thermalInit(void)
+{
+	for (size_t MCSIndex = 0; MCSIndex < MCS_PER_PROC; ++MCSIndex) {
+		for (size_t MCAIndex = 0; MCAIndex < MCA_PER_MCS; ++MCAIndex) {
+				mca_and_or(
+					mcs_to_nest[mcs_ids[MCSIndex]], MCAIndex, MCA_MBA_FARB3Q,
```

Comment:
```suggestion
					mcs_ids[MCSIndex], MCAIndex, MCA_MBA_FARB3Q,
```
These are MCA registers, they are not under Nest chiplet.